examples/storybook/src/wrap-with-hits.ts

```diff
@@ -15,6 +15,7 @@ type WrapWithHitsParams = {
   filters?: string;
   indexName?: string;
   hits?: string;
+  routing?: {};
```

Comment:
If we respect the InstantSearch usage, it can also be a boolean.

```suggestion
  routing?: boolean | {};
```